pkgs/by-name/ll/llama-cpp/package.nix

```
@@ -94,6 +96,7 @@ effectiveStdenv.mkDerivation (finalAttrs: {
     ++ optionals mpiSupport mpi
     ++ optionals openclSupport [ clblast ]
     ++ optionals rocmSupport rocmBuildInputs
+    ++ optional blasSupport openblas.dev
```

Let's use `optionals` here.

It's true, it's opinionated. But `optional` vs `optionals` is always an issue for newcomers.

`optional` implicitly wrap the value in a list, while `optionals` doesn't. `optionals` have a lower cognitive load than `optional`, and I think that we should use it here.

```suggestion
    ++ optionals blasSupport [ openblas.dev ]
```

Another thing: I also notice that the derivation `mpi` (line 96) should also be written `[ mpi ]`.

---

thanks, I updated the PR